<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>601</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tmon몬소리OTF Black</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Break Captcha</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="userLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>591</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string>사용자 정보 : 미설정</string>
   </property>
  </widget>
  <widget class="QLabel" name="chromeLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>591</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string>크롬드라이버 : 미설정</string>
   </property>
  </widget>
  <widget class="QPushButton" name="personInfoSetButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>경기천년제목V Bold</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>사용자 정보 설정</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openChromeButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>경기천년제목V Bold</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>크롬 실행</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="chromeSetButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>경기천년제목V Bold</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>크롬드라이버 설정</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aboutButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>경기천년제목V Bold</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>경기천년제목V Bold</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>601</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>601</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>personInfoSetButton</tabstop>
  <tabstop>openChromeButton</tabstop>
  <tabstop>chromeSetButton</tabstop>
  <tabstop>aboutButton</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
